docs/Tutorials/submit-tutorial.md

```diff
@@ -1,19 +1,34 @@
 ---
-title: Submit Tutorial
-id: submit-tutorial
+title: Submit a Tutorial
+id: submit-a-tutorial
 ---
 
-If you've got an idea for a Tutorial then go ahead and submit an issue. 
+### 💡 Looking to contribute your tutorial to the [Ankr Docs](https://ankr.com/docs/) using our stack? 👀
+🏗 Join our community of builders and contribute to the open-source development of Ankr Protocol by accelerating the education of Web3 builders.
 
-[Submit Tutorial Idea](https://github.com/Ankr-network/ankr-docs/issues/new?assignees=&labels=&template=propose-new-content.md&title=)
-
-Complete the issue form and request a review.
-
-The Community Tutorials team can then check out your idea and if it fulfils our criteria we'll encourage you to create a tutorial.
+🧾 Open-source software without documentation will be the Achilles' heel of Web3. If you see an issue or edit that needs made, create a pull request on the [🔃 Ankr Docs Github](https://github.com/Ankr-network/ankr-docs).
```

Comment:
Can we tweak this to be a bit more positive e.g. "Great docs are the future of an inclusive Web3.  Be a part of the movement". Also, unfortunately, our software is not open source. There are open source parts to it but it's not open source!! Otherwise, we could have got a free search with Doc Search.

Reply:
Sounds good, yep. Want to put your twist on it?